From d8c9633ffb7679abbf8748f6967a2adf7ff7309c Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Wed, 12 Dec 2012 14:48:18 +0800
Subject: [PATCH 115/304] ENGR00236052 add bug mailer in product files.

add bug mailer, if use can get correct send_bug.jar.
The "Report Bug" function will work on Setting->Development

Not found a corrent send_bug.jar yet.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 imx6/etc/init.rc | 7 +++++++
 imx6/imx6.mk     | 7 +++++++
 2 files changed, 14 insertions(+)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index ec0b73a..a384792 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -554,6 +554,13 @@ service dumpstate /system/bin/dumpstate -s
     disabled
     oneshot
 
+# bugreport is triggered by the VOLUME-DOWN and VOLUME-UP keys
+service bugreport /system/bin/bugmailer.sh -b /system/media/audio/ui/Dock.ogg -e /system/media/audio/ui/Undock.ogg
+    class main
+    disabled
+    oneshot
+    keycodes 115 114
+
 service sshd /system/bin/start-ssh
     class main
     disabled
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 2e6c67f..830c66e 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -286,6 +286,13 @@ PRODUCT_COPY_FILES +=	\
 	device/fsl/common/display/display_mode_fb2.conf:system/etc/display_mode_fb2.conf \
 	device/fsl/common/display/display_mode_fb4.conf:system/etc/display_mode_fb4.conf
 
+# for bugmailer
+PRODUCT_PACKAGES += send_bug
+PRODUCT_COPY_FILES += \
+    system/extras/bugmailer/bugmailer.sh:system/bin/bugmailer.sh \
+    system/extras/bugmailer/send_bug:system/bin/send_bug
+
+
 PRODUCT_COPY_FILES +=	\
 	external/linux-firmware-imx/firmware/vpu/vpu_fw_imx6d.bin:system/lib/firmware/vpu/vpu_fw_imx6d.bin 	\
 	external/linux-firmware-imx/firmware/vpu/vpu_fw_imx6q.bin:system/lib/firmware/vpu/vpu_fw_imx6q.bin
-- 
1.8.0

